### PR TITLE
[CICD-883] Update @changesets/cli to 2.27.10

### PR DESCRIPTION
**Pull request by mike-day**
# JIRA Ticket

[CICD-833](https://wpengine.atlassian.net/browse/CICD-833)

## What Are We Doing Here

Sister PR to https://github.com/wpengine/site-deploy/pull/41, updating the same deps.

Resolves a vulnerability with a transitive dependency of @changesets/cli, which was patched in [@changesets/cli 2.7.10](https://github.com/changesets/changesets/releases/tag/%40changesets%2Fcli%402.27.10#:~:text=%231514%20962ab91%20Thanks%20%40nicoalonsop!%20%2D%20Update%20spawndamnit%20to%20fix%20cross%2Dspawn%20vulnerability).

Since @changesets/cli is a dev dependency for the repo, risk of releasing this change is low.

---

Offending transitive dependency tree before and after:

Screenshot 2024-12-04 at 11 46 45 AM
Screenshot 2024-12-04 at 11 46 33 AM

[CICD-833]: https://wpengine.atlassian.net/browse/CICD-833?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md
Comment (apmatthews) [inline README.md:1]: Looks like this was added in commit 01b5b175e690e430bc3344edd20289bb8a853e6c. Did that somehow sneak into this branch?

Comment (mike-day) [reply README.md:1]: Great catch; I must've based this PR on the wrong branch 🙃

Just rebased onto `main` and pushed up, all should be good now!

Comment (mike-day) [reply README.md:1]: I take it back, there is a failing test now 👀

Comment (mike-day) [reply README.md:1]: Hmm:

```
Unable to find image 'wpengine/site-deploy:1.0.4' locally
docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
```

Comment (apmatthews) [reply README.md:1]: I've seen that happen occasionally. Not entirely sure how we'd be hitting a rate limit, but you should be able to re-run.

**Review by apmatthews — APPROVED** [review]

**Pull request merged**